api-schema/xdb.yaml

```diff
@@ -84,7 +84,31 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ApiErrorResponse'
-  # xdb workers APIs are hosted by iWF applications via SDKs, for iWF server to call back
+  /api/v1/xdb/service/process-execution/send-message-to-local-queue:
```

Comment:
As we said the name is too long, I have been thinking the name. Actually in iwf, it's called `publish_to_internal_channel`. Maybe we can also call it `publish_to_local_queue` here? which one do you think is "slightly" better?

Reply:
`publish-to-local-queue` sounds good to me!